docs/streams/developer-guide/config-streams.html

```diff
@@ -284,6 +284,11 @@ <h4><a class="toc-backref" href="#id5">bootstrap.servers</a><a class="headerlink
             <td colspan="2">Timestamp extractor class that implements the <code class="docutils literal"><span class="pre">TimestampExtractor</span></code> interface.</td>
             <td>See <a class="reference internal" href="#streams-developer-guide-timestamp-extractor"><span class="std std-ref">Timestamp Extractor</span></a></td>
           </tr>
+          <tr class="row-odd"><td>upgrade.from</td>
```

Comment:
@ableegoldman minor nit: row-odd => row-even.
also need to update below rows

Reply:
https://github.com/apache/kafka/pull/7837

Reply:
We'll have to cherry-pick this PR to 2.4 first, then we can merge/cherry-pick that fix on top of it